tools/wasme/cli/docs/content/tutorial_code/wasme_operator.md

```diff
@@ -192,13 +191,16 @@ Note the `status` of the FilterDeployment
 apiVersion: wasme.io/v1
```

Comment:
Since we added lines here, we'll have to update `hl_lines` (Highlight lines) numbers to reflect the new lines we want to highlight. In this instance we're highlighting the `Status` field and it's children.

Reply:
done